src/kernel.jl

```diff
@@ -380,6 +388,40 @@ function Base.convert(::Type{AbstractArray}, L::Laplacian{N}) where N
 end
 _reshape(L::Laplacian{N}, ::Val{N}) where {N} = L
 
+"""
+    laplacian2d(alpha::Number)
+
+Construct a weighted discrete Laplacian approximation in 2d. `alpha` controls the weighting of the faces
+relative to the corners.
+
+# Examples
+
+```jldoctest
+julia> Kernel.laplacian2d(0)      # the standard Laplacian
+3×3 OffsetArray(::Matrix{Float64}, -1:1, -1:1) with eltype Float64 with indices -1:1×-1:1:
+ 0.0   1.0  0.0
+ 1.0  -4.0  1.0
+ 0.0   1.0  0.0
+
+julia> Kernel.laplacian2d(1)      # a corner-focused Laplacian
+3×3 OffsetArray(::Matrix{Float64}, -1:1, -1:1) with eltype Float64 with indices -1:1×-1:1:
+ 0.5   0.0  0.5
+ 0.0  -2.0  0.0
+ 0.5   0.0  0.5
+
+julia> Kernel.laplacian2d(0.5)    # equal weight for face-pixels and corner-pixels.
+3×3 OffsetArray(::Matrix{Float64}, -1:1, -1:1) with eltype Float64 with indices -1:1×-1:1:
+ 0.333333   0.333333  0.333333
+ 0.333333  -2.66667   0.333333
+ 0.333333   0.333333  0.333333
+```
+"""
+function laplacian2d(alpha::Number=0)
+    lc = alpha/(1 + alpha)
```

Comment:
Should `alpha > -1` here?

Reply:
Certainly `alpha = -1` is a bad choice. But I can't say that
```julia
julia> Kernel.laplacian2d(-2)
3×3 OffsetArray(::Matrix{Float64}, -1:1, -1:1) with eltype Float64 with indices -1:1×-1:1:
  2.0  -3.0   2.0
 -3.0   4.0  -3.0
  2.0  -3.0   2.0
```
would never be useful to anyone. And it has some laplacian-like properties, such as a sum of 0, symmetry, and satisfaction of Green's-like identities. 

I wish this had come with a reference, so we could check a citation, but believe it or not I think this predates my own 2012 involvement with Julia!